### PR TITLE
UI redesign

### LINKED ISSUE #34: New design
It has to follow [Habitica rules](https://habitica.fandom.com/wiki/Guidance_for_Comrades#Rules_for_Third-Party_Tools), but I would like to have something more modern and less white. Though I like the cards thing.

### PR DESCRIPTION
**Pull request by nebocoder**
### Closes #34.

Hope it isn't an issue that HTML got formatted a bit differently. I had to add some classes and also add a missing opening body tag in order for my CSS to work.

Screenshot:
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,33 +1,65 @@
...
+          the
+          <a href="https://github.com/Ieahleen/habitican-randomizer"
+            >GitHub Repository</a
+          >
+          or in the
+          <a
+            href="https://habitica.com/groups/guild/e4956785-9b42-470a-b1af-18d9c5e24769"
+            >Habitica Guild</a
+          >.</p
+        >
```
Comment (ieahleen) [inline index.html:34]: Would you mind moving this in the footer?

**Review by ieahleen — CHANGES_REQUESTED** [review]
It's going really well, thank you! One small thing

File: index.html (hunk 1 above)
Comment (nebocoder) [reply index.html]: Done!

**Review by ieahleen — COMMENTED** [review]
Thank you for this PR!

**Pull request merged**